bedrock/contentful/constants.py

```diff
@@ -9,11 +9,13 @@
 CONTENT_TYPE_PAGE_RESOURCE_CENTER = "pagePageResourceCenter"
 CONTENT_TYPE_PAGE_GENERAL = "pageGeneral"
 CONTENT_TYPE_PAGE_VERSATILE = "pageVersatile"
+CONTENT_TYPE_PAGE_PRODUCT_STORY = "pageProductJournalismStory"
 
 DEFAULT_CONTENT_TYPES = ",".join(
     [
         CONTENT_TYPE_CONNECT_HOMEPAGE,  # The Connect-based approach, currently used for the homepage
         CONTENT_TYPE_PAGE_RESOURCE_CENTER,  # New-era Compose page with a dedicated type
+        CONTENT_TYPE_PAGE_PRODUCT_STORY,  # Compose page for PBJ Stories
```

Comment:
Q: do we want to rename `PBJ` to `PJB`/`PJS` throughout?

Reply:
I'll stick with Product Story for consistency, it was easier for me to remember PBJ as an internal name and mnemonic device, but PJB or PJS is a new acronym to remember, and we have enough of those